src/http/server/handlers/log_handler.cr

```diff
@@ -1,20 +1,18 @@
 # A handler that logs the request method, resource, status code, and
-# the time used to execute the next handler, to the given `IO`.
+# the time used to execute the next handler, to the log with source `http.server`.
 class HTTP::LogHandler
   include HTTP::Handler
-
-  # Initializes this handler to log to the given `IO`.
-  def initialize(@io : IO = STDOUT)
```

Comment:
This is a breaking-change.

Ideally we could add a `@Deprecated` to a `def initialize(io : IO)` constructor even if we are no longer using that `io`.

As you prefer.

Reply:
Done

Reply:
Not done... I broke everything 🙄

Reply:
There should be an initializer accepting a `Log` instance so users can use a custom configured one. The legacy initializer accepting `IO` could be implemented that way, creating a custom `Log` instance that logs to the given io.

Reply:
I don't really mind so much whether it accepts a `Log`, but it must at least accept the source string.